Framework/DataHandling/src/LoadMcStas.cpp

```diff
@@ -269,25 +277,33 @@ std::vector<std::string> LoadMcStas::readEventData(
   double shortestTOF(0.0);
   double longestTOF(0.0);
 
+  // create vector container all the event output workspaces needed
   const size_t numEventEntries = eventEntries.size();
   std::string nameOfGroupWS = getProperty("OutputWorkspace");
   const auto eventDataTotalName = std::string("EventData_") + nameOfGroupWS;
   std::vector<std::pair<EventWorkspace_sptr, std::string>> allEventWS = {
       {eventWS, eventDataTotalName}};
+  // if numEventEntries > 1 also create separate event workspaces
+  const bool onlySummedEventWorkspace =
+      getProperty("OutputOnlySummedEventWorkspace");
+  if (!onlySummedEventWorkspace && numEventEntries > 1) {
+    for (const auto &eventEntry : eventEntries) {
+      const std::string &dataName = eventEntry.first;
+      // create container to hold partial event data
+      // plus the name users will see for it
+      const auto ws_name = dataName + std::string("_") + nameOfGroupWS;
```

Comment:
I don't think `std::string` is needed here - isn't `dataname + "_" + nameOfGroupWS` the same?